buildkite/build_project.sh

```diff
@@ -201,6 +201,14 @@ case "$REPO_FULL_NAME" in
         # remove once https://github.com/rejectedsoftware/ddox/pull/212 is merged and released
         sed 's/bridgeip.*/exit \$failure/' -i travis-ci.sh # build without the phantomcss tester
         ;;
+
+    ldc-developers/ldc)
+        git submodule update --init
```

Comment:
Would be nice to not download the stdlib unnecessarily every time: does replacing this with `rm runtime/CMakeLists.txt` work instead?

Reply:
Nope. I tried it myself after seeing this PR (plain ldc build without submodules), almost got it to work eventually but then dismissed the idea (Idon't recall the actual reason anymore though, but it didn't seem worth it).